src/printer.js

```diff
@@ -387,10 +387,22 @@ const embed = (path, print, textToDoc, opts) => {
   }
 
   if (node.type === 'Style') {
+    let styleLang = '';
+    let parserLang = '';
+
+    if ('attributes' in node) {
+      const langAttribute = node.attributes.filter((x) => x.name === 'lang');
+      if (langAttribute.length === 0) styleLang = 'css';
+      else {
+        styleLang = langAttribute[0].value[0].raw.toLowerCase();
```

Comment:
Why access the raw form here? 🤔

Reply:
Because it contained the value of the attribute. Not sure if there's a better way, because I've never worked on Prettier plugins.
I know its a bit obscure, maybe I should add a comment on why I did it.

Reply:
The value of `langAttribute` is like this:
```js
{ start: 336, end: 340, type: 'Text', raw: 'scss', data: 'scss' }
```
That is why I've done that

Reply:
Ah, I think 'data' might be more correct? idk...

Reply:
Maybe yes. We can always swap it out if we encounter something wrong